URBNSwiftAlert/Classes/AlertConfiguration.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AlertConfiguration.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 5/22/17.
+//
+//
+import Foundation
+
+public struct AlertConfiguration {
+    /**
+     *  Title text for the alert
+     */
+    public var title: String?
+    
+    /**
+     *  Message text for the alert
+     */
+    public var message: String?
+    
+    /**
+     *  The view to present from when using showInView:
+     */
+    public var presentationView: UIView?
+    
+    /**
+     *  Flag if the alert is active. False = a passive alert
+     */
+    public var isActiveAlert = false
```

Comment:
Why are we defaulting to passive alert?

Reply:
we don't consider an alert to be active unless the implementer has added an active action (adds a button) to the alert.  alerts don't have a default button added, so they're passive by default.

Reply:
changed to a computed var